spec/std/isa/csr/sstatus.yaml

```diff
@@ -93,6 +93,16 @@ fields:
     type: RW-H
     reset_value: UNDEFINED_LEGAL
     definedBy: V
+  SDT:
+    alias: mstatus.SDT
```

Comment:
Aliases don't (currently) inherit anything from the aliasee, so please copy the `sw_write()` method from `mstatus.SDT` here.